sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/OperatorStateMetadataSuite.scala

```diff
@@ -145,14 +142,37 @@ class OperatorStateMetadataSuite extends StreamTest with SharedSparkSession {
       StopStream
     )
 
-    val statePath = new Path(checkpointDir.toString, "state/0")
-    val operatorMetadata = new OperatorStateMetadataReader(statePath, hadoopConf).read()
-      .asInstanceOf[OperatorStateMetadataV1]
-
     val expectedMetadata = OperatorStateMetadataV1(
       OperatorInfoV1(0, "sessionWindowStateStoreSaveExec"),
       Array(StateStoreMetadataV1("default", 1, spark.sessionState.conf.numShufflePartitions))
     )
-    assert(sameOperatorStateMetadata(operatorMetadata, expectedMetadata))
+    checkOperatorStateMetadata(0, expectedMetadata)
+  }
+
+  test("Stateful operator metadata for multiple operators.") {
```

Comment:
nit: remove `.` at the end for consistency

Reply:
Removed.